Formula/sstp-client.rb

```diff
@@ -1,10 +1,9 @@
 class SstpClient < Formula
   desc "SSTP (Microsofts Remote Access Solution for PPP over SSL) client"
   homepage "https://sstp-client.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/sstp-client-1.0.13.tar.gz"
-  sha256 "961258fca0795d8ad60b047942cf7cb53d025d353fd1e4ba08c2b75799f5321b"
+  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/sstp-client-1.0.15.tar.gz"
+  sha256 "8484aa51fbfbe418a0ebad58ad20a8ee1c46ed71f800be18bcd23b42e6baad64"
   license "GPL-2.0-or-later"
-  version_scheme 1
```

Comment:
Please don't remove this.

Reply:
Hm, I thought this was only necessary for the downgrade from 1.0.14 to 1.0.13, therefore I removed that. I will submit another commit to reintroduce this option.